### ISSUE TITLE
Extraction with robot not working

### ISSUE THREAD

**Issue by luisenriqueramos1977** [HajoRijgersberg/OM#64]
Dear All, 

I am trying to extract some concepts of this ontology using robot. I found two files in this repository, those are: om-2-ucum.ttl  and om-2.0.rdf.
 
With them, I edited the following commands:
robot extract --method STAR --input source_ontos/om-2-ucum.ttl --term-file onto_terms/om2_terms.txt --output extracted_modules/om2_start.owl
robot extract --method STAR --input source_ontos/om-2.0.rdf --term-file onto_terms/om2_terms.txt --output extracted_modules/om2_start.owl

Obtaining the following results:

![image](https://user-images.githubusercontent.com/48624384/163329998-ba615e3d-80c7-4e03-8e9c-c73c1b621078.png)

After that, I decided to save both ontologies as owl files, using protege, but the result of this command:

robot extract --method STAR --input source_ontos/om-2-ucum.owl --term-file onto_terms/om2_terms.txt --output extracted_modules/om2_start.owl

 was exactly the same as the first picture, and the result of applying the extraction command on file om-2.0.owl was the following:

![image](https://user-images.githubusercontent.com/48624384/163330329-86d93738-bec1-41f0-a1e8-58849cd7ba60.png)

to summarize robot (as protege) does not find any ontology in:

http://www.ontology-of-units-of-measurement.org/resource/om-2/

So, I wonder how to solve this issue?

I am also curious to know if you have an .owl version of this ontology, because I only found .ttl and .rdf.

Best regards

Luis Ramos

**Comment by HajoRijgersberg** [comment]
Hi Luis,
Thanx for you issue.
First of all: the link you use (http://www.ontology-of-units-of-measurement.org/resource/om-2/) is not correct. It should be: http://www.ontology-of-units-of-measure.org/resource/om-2/
Second, the screenshots of your error messages unfortunately are unreadable. Could you perhaps give the messages in some other way or a using sharper screenshots?
Thanx in advance, and of course I hope that my first point already helps! :)
Cheers, Hajo

**Comment by BonaBeavis** [comment]
The underlying issue could be that it is not possible to retrieve the ontology without a browser:

<pre><font color="#FF0086">❯</font> <font color="#00C918"><b>~</b></font> wget  http://www.ontology-of-units-of-measure.org/resource/om-2/
--2022-05-22 14:11:08--  http://www.ontology-of-units-of-measure.org/resource/om-2/
Resolving www.ontology-of-units-of-measure.org (www.ontology-of-units-of-measure.org)... 137.224.11.21
Connecting to www.ontology-of-units-of-measure.org (www.ontology-of-units-of-measure.org)|137.224.11.21|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: /data/om-2 [following]
--2022-05-22 14:11:08--  http://www.ontology-of-units-of-measure.org/data/om-2
Reusing existing connection to www.ontology-of-units-of-measure.org:80.
HTTP request sent, awaiting response... 500 Internal Server Error
2022-05-22 14:11:08 ERROR 500: Internal Server Error.
</pre>

**Comment by HajoRijgersberg** [comment]
@luisenriqueramos1977 , sorry I overlooked your last question:

> I am also curious to know if you have an .owl version of this ontology, because I only found .ttl and .rdf.

No we only have an rdf version.

**Comment by HajoRijgersberg** [comment]
@BonaBeavis , thanx for your comment.
Not sure how we can solve this... Have to think about it, ask round.
What happens if you would use http://www.ontology-of-units-of-measure.org/page/om-2/   ?
Best, Hajo

**Comment by BonaBeavis** [comment]
With `http://www.ontology-of-units-of-measure.org/page/om-2/` wget only gets a html page.

Here is a guide how to solve this: https://www.w3.org/TR/swbp-vocab-pub/

**Comment by HajoRijgersberg** [comment]
Thanx for trying. Indeed, that's not the result we want.
I'll raise the topic to a colleague of mine, along with the valuable link you give.
To be continued.

**Comment by HajoRijgersberg** [comment]
A colleague of mine is looking for ways to organizing solving this problem.
We are also discussing the problem itself. Could you perhaps try http://foodvoc.org/resource/om-2?

**Comment by BonaBeavis** [comment]
Same thing.

<pre><font color="#FF0086">❯</font> <font color="#00C918"><b>d</b></font> wget http://foodvoc.org/resource/om-2
--2022-05-25 16:51:00--  http://foodvoc.org/resource/om-2
Resolving foodvoc.org (foodvoc.org)... 137.224.11.21
Connecting to foodvoc.org (foodvoc.org)|137.224.11.21|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: /data/om-2 [following]
--2022-05-25 16:51:01--  http://foodvoc.org/data/om-2
Reusing existing connection to foodvoc.org:80.
HTTP request sent, awaiting response... 500 Internal Server Error
2022-05-25 16:51:01 ERROR 500: Internal Server Error.
</pre>

**Comment by HajoRijgersberg** [comment]
Thanx. One never knows. Unfortunately it didn't help.
To be continued with my colleague.
